pom.xml

```diff
@@ -19,7 +19,7 @@
 		<scala.version>2.11.8</scala.version>
 		<scala.binary.version>2.11</scala.binary.version>
 		<scoverage.plugin.version>1.3.0</scoverage.plugin.version>
-		<rug.version>1.0.0-m.3</rug.version>
+		<rug.version>1.0.0-20170530101352</rug.version>
```

Comment:
I sometime see @cdupuis doing this. It's just a a case of making sure we change them to releases versions before releasing I think.